Jenkinsfile

```diff
@@ -49,14 +49,50 @@ pipeline {
         // }
       }
     }
-    stage('Documentation') {
-      steps {
-        sh '''
-          make deps-media
-          make .build/mkr-mcd-rtd.tar
-          cp .build/mkr-mcd-rtd.tar ./
-        '''
-        archiveArtifacts artifacts: 'mkr-mcd-rtd.tar'
+    stage('Deploy') {
+      when { branch 'master' }
+      post {
+        failure {
+          slackSend color: '#cb2431'                            \
+                  , channel: '#maker-internal'                  \
+                  , message: "Deploy failure: ${env.BUILD_URL}"
+        }
+      }
+      stages {
+        stage('Initialize Git/SSH') {
+          steps {
+            sshagent(['2b3d8d6b-0855-4b59-864a-6b3ddf9c9d1a']) {
+              sh '''
+                git config --global user.email "admin@runtimeverification.com"
+                git config --global user.name  "RV Jenkins"
+                mkdir -p ~/.ssh
+                echo 'host github.com'                       > ~/.ssh/config
+                echo '    hostname github.com'              >> ~/.ssh/config
+                echo '    user git'                         >> ~/.ssh/config
+                echo '    identityagent SSH_AUTH_SOCK'      >> ~/.ssh/config
+                echo '    stricthostkeychecking accept-new' >> ~/.ssh/config
+                chmod go-rwx -R ~/.ssh
+                ssh github.com || true
```

Comment:
What is happening at the end of this stage? Is the next stage being ran from an ssh session?

Reply:
No, this logs into GitHub a single time using the authentication to setup the `known_hosts` file.